core/src/main/scala/org/bitcoins/core/gcs/GCS.scala

```diff
@@ -48,11 +48,14 @@ object GCS {
     */
   def buildBasicBlockFilter(
       data: Vector[ByteVector],
-      key: ByteVector): GolombFilter = {
+      key: SipHashKey): GolombFilter = {
     buildGolombFilter(data, key, BlockFilter.P, BlockFilter.M)
   }
 
-  private def sipHash(item: ByteVector, key: ByteVector): UInt64 = {
+  //item remains bytevector key to be changed create new file in GCS folder that redefines sipkeyhash
+  //will probably have a require statement bytevector length of 16 bytes or 128 bits. Add a couple methods
+  //to turn the key into an array of bytes (look to toArray method on line 65)
+  private def sipHash(item: ByteVector, key: SipHashKey): UInt64 = {
```

Comment:
nit: Generally above `def`s its best to wrap comments with
```
/**
  * Comment
  * Here
  */
```
since that auto-generates scaladocs :)

Reply:
nit: Also in general its best to have the comment above a def describe what the def does and what its non-trivial inputs are so that the comment stays relevant in the future, rather than describing changes to the def